### ISSUE TITLE
validation set

### ISSUE THREAD

**Issue by ireneb612** [SHI-Labs/Compact-Transformers#59]
Hi!
I started training your model on my data, and after some reserach I did not manage to understand how the validation set should be made in terms of directories! I know it's timm library, but I was wondering if you could help me.

My training data il like this: -/train/0/class1/ .....jpeg
                                                           class2/ .....
                                                           class3/....
                                                           class4/....

How and where should I position my validation folder?

**Comment by alihassanijr** [comment]
Hello,

The default is `val/`, but it can be overridden using the `--val-split` argument.
